zebra-chain/src/treestate/note_commitment_tree.rs

```diff
@@ -9,14 +9,53 @@
 //! append-only.
 //!
 //! A root of a note commitment tree is associated with each treestate.
+
 #![allow(clippy::unit_arg)]
```

Comment:
same unit_arg concern as usual

Reply:
This one is needed or clippy complains
![image](https://user-images.githubusercontent.com/552961/89630793-d5e7e200-d86d-11ea-86fe-e474acd0bf17.png)